flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveTableSource.java

```diff
@@ -156,7 +156,9 @@ public boolean isBounded() {
                                             HiveSourceFileEnumerator.createInputSplits(
                                                             0,
                                                             hivePartitionsToRead,
-                                                            flinkConf,
+                                                            flinkConf.get(
+                                                                    HiveOptions
```

Comment:
We need it, since it's passed in `HiveParallelismInference` and `HiveSourceBuilder` (the latter is `@PublicEvolving` and we'd need to change the ctor)